tests/flow/binding/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -450,7 +450,7 @@ function type_shadow_nested_scope() {
 
 // fn params name clash
 
-function fn_params_name_clash(x, x /* error: x already bound */) {}
+function fn_params_name_clash(/* error: x already bound */ x, x) {}
```

Comment:
This is a regression. I'm not sure what this PR is trying to fix, although TBH I haven't read the original issue yet. The previous comment placement (after the var) is correct.